instance-scheduler/main_test.go

```diff
@@ -187,6 +189,412 @@ func (m *mockIEC2InstancesAPI) StartInstances(ctx context.Context, params *ec2.S
 	return m.StartInstancesOutput, nil
 }
 
+type mockIRDSInstancesAPI struct {
+	DescribeDBInstancesOutput *rds.DescribeDBInstancesOutput
+	StartDBInstanceOutput     *rds.StartDBInstanceOutput
+	StopDBInstanceOutput      *rds.StopDBInstanceOutput
+}
+
+func (m *mockIRDSInstancesAPI) DescribeDBInstances(ctx context.Context, params *rds.DescribeDBInstancesInput, optFns ...func(*rds.Options)) (*rds.DescribeDBInstancesOutput, error) {
+	return m.DescribeDBInstancesOutput, nil
+}
+
+func (m *mockIRDSInstancesAPI) StopDBInstance(ctx context.Context, params *rds.StopDBInstanceInput, optFns ...func(*rds.Options)) (*rds.StopDBInstanceOutput, error) {
+	return m.StopDBInstanceOutput, nil
+}
+
+func (m *mockIRDSInstancesAPI) StartDBInstance(ctx context.Context, params *rds.StartDBInstanceInput, optFns ...func(*rds.Options)) (*rds.StartDBInstanceOutput, error) {
+	return m.StartDBInstanceOutput, nil
+}
+func TestStopStartTestRDSInstancesInMemberAccount(t *testing.T) {
+	tests := []struct {
+		testTitle     string
+		client        *mockIRDSInstancesAPI
+		action        string
+		expectedCount RDSInstanceCount
+	}{
+		{
+			testTitle: "RDS testing Stop action",
```

Comment:
You say you are testing Stop action, but you pass "Test" action. 
The count will differ for different actions, I will assume this meant to be Stop action

Reply:
this is a mislabelling changed to test